README.md

```diff
@@ -28,6 +28,9 @@ http://entrproject.org/) utility:
 ```bash
 find . -name "*_shpec.sh" | entr shpec
 ```
+__Warning__: If you use `zsh` you should disable the `end` keyword. (to do that, run `disable -r end`.
```

Comment:
This is such a unique situation that I hesitate to include it in the README; it would be noise for the majority of readers. Perhaps we could simply check for zsh inside of shpec, and do this `disable` ourselves if necessary.